docs/feature/storage/index.md

```diff
@@ -89,15 +67,94 @@ Inverted indexes for text values, BKD trees for numeric values, and doc values.
   all field values that are not analyzed as strings in a compact column, making it more
   effective for sorting and aggregations.
 
-## See also
+:Column store:
+
+  CrateDB implements a {ref}`column store <crate-reference:ddl-storage-columnstore>`
+  based on doc values in Lucene.
+  For text values, other than storing the row data as-is (and indexing each value by default),
+  each value term is stored into a column-based store by default.
+
+  This storage layout improves the performance of sorting, grouping, and aggregations,
+  by keeping field data for one column packed at one place rather than scattered across documents.
+  The column store is enabled by default in CrateDB and can be disabled only for text fields.
```

Comment:
It can be disabled for other types as well, see https://cratedb.com/docs/crate/reference/en/latest/general/ddl/storage.html#column-store.